quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -62,71 +53,53 @@ object VitaminQuantityPickers {
         subtractEnabled: Boolean = true,
         editTextEnabled: Boolean = true,
         isExpanded: Boolean = false,
+        helperText: String? = null,
         keyboardOptions: KeyboardOptions = KeyboardOptions(keyboardType = KeyboardType.Number),
         keyboardActions: KeyboardActions = KeyboardActions.Default,
         colors: QuantityColors = VitaminQuantitiesColors.normal(),
-        shape: CornerBasedShape = VitaminTheme.shapes.radius100,
-        textStyle: TextStyle = VitaminTheme.typography.text2,
+        sizes: QuantitySizes = VitaminQuantitiesSizes.primary(),
         ripple: RippleTheme = VitaminTheme.ripples.brand,
         onAddClicked: () -> Unit,
         onSubtractClicked: () -> Unit,
         onValueChange: (String) -> Unit
     ) {
         CompositionLocalProvider(LocalRippleTheme provides ripple) {
-            Row(
+            val contentDescription = stringResource(id = R.string.vtmn_quantity_picker_accessibility_message, value)
+
+            Column(
                 modifier = modifier
-                    .height(48.dp)
-                    .then(
-                        if (isExpanded) {
-                            modifier.fillMaxWidth()
-                        } else {
-                            modifier.width(156.dp)
-                        }
-                    )
+                    .semantics(
```

Comment:
I'm suprized you don't use `error` call inside semantic callback.
It means that you don't inform blind user when the quantity picker is in error?

Reply:
Done

https://user-images.githubusercontent.com/46709142/227160080-86124c47-85a2-4c4b-bfaf-48aae25c09a8.mp4